net-misc/python-x2go/python-x2go-0.6.1.3.ebuild

```diff
@@ -13,10 +13,10 @@ SRC_URI="http://code.x2go.org/releases/source/${PN}/${P}.tar.gz"
 LICENSE="AGPL-3"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
 # TODO: minimal USE flag in net-misc/nx, we only need nxproxy/nxcomp
```

Comment:
Definitely needed at build time?

Reply:
Hmm, [Arch Linux](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=python2-x2go#n12) mentions them at build time as well

Reply:
Compiling something is the main reason that stuff is a genuine dep, or maybe something gets generated. If it's normal setup.py or just installing Python files, no need.

What's it do at build time?

Reply:
:+1:  It's installing python files, so I'll drop DEPEND